webform_strawberryfield.routing.yml

```diff
@@ -19,7 +19,7 @@ webform_strawberryfield.auth_autocomplete:
   requirements:
     _access: 'TRUE'
 webform_strawberryfield.nominatim:
-  path: '/webform_strawberry/nominatim/{api_type}/{count}/{lang}'
+  path: '/webform_strawberry/nominatim/{api_type}/{count<\d+>?5}/{lang}'
```

Comment:
This is cool! but for the sake of easier reading. Maybe its better to leave {count} as such and the add the digit constraint to a requirement like https://www.drupal.org/docs/8/api/routing-system/parameters-in-routes/validate-route-parameters and also give {count} a default of 1? What do you think? Could you do that and make sure the route is working afterwards? Thanks a lot. You are almost there, 96%!

Reply:
@DiegoPino Ok, this is taken care of. When testing I realized an optional param has to go at the end of the route (or be followed by other optional params...duh). So now it is `/{api_type}/{lang}/{count}` with count as default 1.